filament/backend/src/opengl/PlatformEGL.cpp

```diff
@@ -112,7 +118,7 @@ Driver* PlatformEGL::createDriver(void* sharedContext) noexcept {
             EGL_BLUE_SIZE,   8,                                 //  6
             EGL_ALPHA_SIZE,  0,                                 //  8 : reserved to set ALPHA_SIZE below
             EGL_DEPTH_SIZE, 24,                                 // 10
-            EGL_RECORDABLE_ANDROID, 1,                          // 12
+            EGL_RECORDABLE_ANDROID, recordable,                 // 12
```

Comment:
nit: Excluding ANGLE does the job too, though only allowing on Android instead (`#if defined(ANDROID)`) makes the most sense to me. (Though this has been nicely refactored upstream, with checks to EGL extensions first, so once we bump our patch doesn't matter much).

Reply:
I decided not to include this nit, due these changes will be discarded when filament bumped.